onnx/version_converter/convert.h

```diff
@@ -545,6 +545,17 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(std::make_unique<AxesAttributeToInput>("ReduceProd", OpSetID(17), OpSetID(18)));
     registerAdapter(std::make_unique<AxesAttributeToInput>("ReduceSumSquare", OpSetID(17), OpSetID(18)));
 
+    /******** 17 -> 18 ********/
+    registerAdapter(std::make_unique<AxesInputToAttribute>("ReduceL1", OpSetID(18), OpSetID(17)));
```

Comment:
It looks like the implementation of `AxesInputToAttribute` does not handle missing optional parameters. But `Axes` is an optional input to the Reduce operators. So, I guess it needs some extension to handle missing optional parameters. See https://github.com/onnx/onnx/pull/5514/files#r1330811269

Reply:
I agree. The link led me to find this issue. I will depend on the other PR to complete before merge this one.

Reply:
merged with PR 5514

Reply:
Merging now to unblock #5613